src/libraries/System.IO.Ports/src/Interop/Unix/Interop.Serial.cs

```diff
@@ -4,17 +4,111 @@
 
 using System;
 using System.IO.Ports;
+using System.Net.Sockets;
 using System.Runtime.InteropServices;
 using Microsoft.Win32.SafeHandles;
 
 internal static partial class Interop
 {
     internal static partial class Serial
     {
+        [Flags]
```

Comment:
those can probably be shared too

Reply:
The sharing in the native and managed versions should be symmetric. Either share them on both sides or on neither side.

Reply:
The error handling code handling was a lot of code (hundreds of lines, with platform specific ifdefs, etc.), so sharing seemed to be a very good idea. 

The other parts are much less code. I do not have strong opinion about whether to share them. If you have reasons for not sharing them, I will be fine with it.